### DISCUSSION TITLE
장바구니 페이지 최적화 관련

### DISCUSSION THREAD

**Discussion by harry21-kr** [munyeol-Yoon/Permu#63] [자유게시판]
현재 장바구니 페이지 내 모든 컴포넌트에서 `useCart` 훅을 사용하고있는데,
이와 관련해 성능에 문제가 있는지 파악 후에 추 후 최적화가 필요할 것 같습니다.

현재는 컴포넌트화에 집중하여 작성하고, MVP 이후 리팩토링을 통한 최적화를 진행하면 될 것 같습니다.

**Comment by munyeol-Yoon** [comment]
좋은 생각입니다. 기간이 빠듯한 만큼 구현에 집중하고 성능개선은 이후에 진행하시죵!
